sceptre/cli/diff.py

```diff
@@ -0,0 +1,163 @@
+import contextlib
+import json
+import sys
+from datetime import datetime
+from io import StringIO
+from pathlib import Path
+
+import cfn_flip
+import click
+from deepdiff import DeepDiff
+
+from sceptre.cli.helpers import catch_exceptions
+from sceptre.context import SceptreContext
+from sceptre.plan.plan import SceptrePlan
+from sceptre.plan.stack_differ import StackDiff, StackConfiguration
+
+
+@contextlib.contextmanager
+def nullcontext(enter_result=None):
+    yield enter_result
```

Comment:
Python 3.7 added this, but since we need to support 3.6 right now, I defined it here. Easy enough...